eng/pipelines/templates/steps/min-max-testing.yml

```diff
@@ -0,0 +1,144 @@
+parameters:
```

Comment:
i just created archetype-sdk-integration-2.yml to see how this will work with merging these steps into the main yaml. Can you review that? I will have to make changes to tests.yml too for sdks that have their own matrix and unfortunately there are 11 of those :(